lib/diagnostic.rb

```diff
@@ -102,18 +117,26 @@ def eat_meat(food)
       puts 'Yuck!'
     end
   end
-end
+
 
 ##
-# your response here
+class Lion < Cat
+  include Carnivorous
+end
+
+def roar(roars)
+  if roars.class = Lion
+    puts "ROAR!"
+  end
+end
 ##
 
 # #
 # What are some of the advantages of using composition (i.e. mixins)
 # over using direct inheritance?
 
 ##
-# your response as a comment here
+Provides namespace and prevents name clashes
 ##
```

Comment:
The real benefit comes in the ability to draw from many different modules instead of inheriting from one source